CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.13.2
+
+
```

Comment:
Is the empty entry expected?

Reply:
No, see https://github.com/packit/prepare-release/issues/9.